### DISCUSSION TITLE
Questions about Project Submission

### DISCUSSION THREAD

**Discussion by zedhaque** [microsoft/AI-Chat-App-Hack#82] [General]
Hello, 

I was hoping to clarify how the project submission timeline aligns with the schedule for the live showcase (2:00 PM PST) event. Specifically, if I were to submit my project at 23:00 PM PST on February 12th, would it still be possible for it to be included in the showcase? Thank you for your assistance with this matter.

I am currently developing my project in a private repository, but I plan to deploy the chat application to a public domain. Given this situation, would I still be eligible to submit my project for consideration? I appreciate your guidance on this matter.

Cheers

**Comment by pamelafox** [comment]
Re showcase: we're happy to show projects that aren't yet formally submitted, just ping the URL or video to us here.

Re private repository: we typically want to look at any code involved to help us judge the hack. Would you be up for giving us access to see the code in some way? (Emailing a zip or adding my Github to the repo)

**Comment by zedhaque** [comment]
Thanks @pamelafox. 
I have submitted my project. I will find a way to share the code for your review soon.
Many thanks for organizing this :)

**Comment by ksut** [comment]
Do I need to have my project deployed and a publicly available URL to submit it?

**Reply by ksut** [reply]
Nvm.. I see it's not required.
